src/lib/tabs/tab-nav-bar/tab-nav-bar.spec.ts

```diff
@@ -64,6 +63,31 @@ describe('MdTabNavBar', () => {
       expect(inkBar.alignToElement).toHaveBeenCalled();
     });
 
+    it('should re-align the ink bar when the tabs list change', () => {
+      const inkBar = fixture.componentInstance.tabNavBar._inkBar;
+
+      spyOn(inkBar, 'alignToElement');
+
+      fixture.componentInstance.tabs = [1, 2, 3, 4];
+      fixture.detectChanges();
+
+      expect(inkBar.alignToElement).toHaveBeenCalled();
+    });
+
+    it('should re-align the ink bar when the tab labels change the width', done => {
```

Comment:
Nothing seemed to be able to trigger the mutation observer (detect changes, flush microtasks, etc). Took a cue from observe-content.spec.ts which uses `done => ()` as well.